### DISCUSSION TITLE
Is v1.16.0 stable?

### DISCUSSION THREAD

**Discussion by lahwaacz** [openucx/ucx#9693] [Q&A] [answered]
There is v1.16.0 [tag](https://github.com/openucx/ucx/tags) which points to the same commit as v1.16.0-rc2 and no corresponding [release](https://github.com/openucx/ucx/releases). What happened?

**Comment by shamisp** [comment, marked as answer]
One of the developers added the tag by mistake. Sorry about this, it has to be delete. 1.16.0 is not an official release yet.
